deployment/build-s3-dist.sh

```diff
@@ -65,6 +65,16 @@ sed -i -e $replace $template_dist_dir/*.template
 replace="s/%%VERSION%%/$3/g"
 echo "sed -i -e $replace $template_dist_dir/*.template"
 sed -i -e $replace $template_dist_dir/*.template
+replace="s/%%CONTACT_TAG%%/$4/g"
+echo "sed -i -e $replace $template_dist_dir/*.template"
```

Comment:
Let me see if I understand, now we should set CONTACT_TAG, for ex., in env vars, along with BUCKET_NAME, SOLUTION_NAME ...

Reply:
Yes

Reply:
(I updated the release doc to include this stack deploy parameters and commands)